src/commands/build/build.ts

```diff
@@ -290,6 +340,72 @@ filename sascode clear;
 `
 }
 
+async function getFileText(
+  fileName: string,
+  filePath: string,
+  serverType: ServerType
+) {
+  const fileExtension = fileName
+    .substring(fileName.lastIndexOf('.') + 1, fileName.length)
+    .toUpperCase()
+  const { content, encoded, maxLineLength } = await getWebFileContent(
+    filePath,
+    fileExtension
+  )
+
+  return `%let filename=${fileName};
+filename filecode temp lrecl=${maxLineLength};
+data _null_;
+file filecode;
+${content}\n
+run;
+${getWebServiceScriptInvocation(serverType, undefined, false, encoded)}
+filename filecode clear;
+`
+}
+
+async function getWebFileContent(filePath: string, type: string) {
+  let lines,
+    encoded = false
+
+  const typesToEncode: string[] = ['ICO', 'PNG', 'JPG', 'JPEG', 'SVG', 'MP3']
```

Comment:
isn't SVG always text?  no harm to encode I guess

Reply:
removed SVG from list